src/diffusers/models/attention.py

```diff
@@ -196,6 +254,7 @@ def forward(
         timestep: Optional[torch.LongTensor] = None,
         cross_attention_kwargs: Dict[str, Any] = None,
         class_labels: Optional[torch.LongTensor] = None,
+        cond_embeds: Optional[torch.FloatTensor] = None,  # TODO - name
```

Comment:
`cond_embeds` is the name from the openmuse code base. Is there a better name for these? They are the projected pooled embeddings

Reply:
I think `cond_embeds` is okay.

Reply:
For SDXL we have a bit of an ambiguous name as well for the pooled embeddings:

https://github.com/huggingface/diffusers/blob/16851efa0f5248b27f86eb69dbc56900d1cc1c94/src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py#L923C17-L923C34

If the projected pooled embeddings are for additional control I think `additional_cond_embeds` might be better as well. I am also not very sure.

Reply:
cc @patil-suraj wdyt?

Reply:
I'd prefer to actually make it a dict as done for the UNet2DCondition here: https://github.com/huggingface/diffusers/blob/6fac1369d0140ebeafdaeeef3f558f21fa3d5108/src/diffusers/models/unet_2d_condition.py#L806

So maybe just `added_cond_kwargs` with a nice keyword "pooled_text_emb"

Reply:
done!